content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -173,8 +173,7 @@ Cluster Domain will be set to `cluster.local` unless
 
 ### Stable Storage
 
-Kubernetes creates one [PersistentVolume](/docs/concepts/storage/persistent-volumes/) for each
-VolumeClaimTemplate. In the nginx example above, each Pod will receive a single PersistentVolume
+Per each StatefulSet triggered Pod Kubernetes creates a PersistentVolumeClaim object for each VolumeClaimTemplates entry defined in the StatefulSet object. In the nginx example above, each Pod will receive a single PersistentVolume
```

Comment:
The grammar in the original issue wasn't fantastic. Try...

```suggestion
For each VolumeClaimTemplate entry defined in a StatefulSet, each Pod receives one PersistentVolumeClaim. In the nginx example above, each Pod receives a single PersistentVolume
```

Reply:
Hi i have addressed the changes PTAL!

Reply:
@celestehorgan Please let me know if any other changes are required in this PR.

Thanks!

Reply:
Hi @jihoon-seo Could you Please take a look here. I have addressed the changes as suggested